### ISSUE TITLE
JAWS 2018 does not announce data tables with 1 row or column

### ISSUE THREAD

**Issue by janellenchew** [FreedomScientific/standards-support#116]
### Summary

JAWS 2018 does not announce data tables that have 1 rows and n columns or n rows and 1 columns. JAWS 17 and 18 do announce them as data tables. Not sure whether this was an intentional update for JAWS 2018 or a bug since the behavior is different than JAWS 17 and 18.

### Expected result

If a table has a <th> header and scope, I assume that JAWS would read this as a data table. I would expect JAWS to announce "table with 1 columns and 5 rows" and to be able to jump to the table with the "t" key.

### Actual result

JAWS 2018 reads this as a layout table.

Note: I also tried adding datatable=1. This worked for JAWS 2018 on Firefox 56 but not 60.

### Example

<table>
<caption>This is the caption</caption>
  <tr><th scope="col">This is the header</th></tr>
  <tr><td>first</td></tr>
  <tr><td>second</td></tr>
  <tr><td>third</td></tr>
  <tr><td>fourth</td></tr>
</table>

https://codepen.io/anon/pen/QVzZzB

#### JAWS version and build number

JAWS Version 2018.1807.8
Also tested on 17.0.2211 and 18.0.2118.
I migrated the same settings to JAWS 2018 from 18.
 
#### Operating System and version

Windows 7

#### Browser and version:

Chrome Version 66.0.3359.181 (Official Build) (32-bit)
Firefox 56.0.2 (32-bit)
Firefox Quantum 60.0.1 (64-bit)

**Comment by MichaelKetting** [comment]
Can also be reproduced with JAWS 2019.1810.64

**Comment by MichaelKetting** [comment]
@stevefaulkner is there any update on this regression issue?

**Comment by stevefaulkner** [comment]
@MichaelKetting have asked @ggordon-vfo to review

**Comment by kurtjwolf** [comment]
Potentially regression issue. Filed internally as bug 104063.
